modules/customizing-the-global-header/proc-customizing-rhdh-global-header.adoc

```diff
@@ -0,0 +1,38 @@
+[id="customizing-your-product-global-header_{context}"]
+= Customizing your {product} global header
+
+To replace the {product} global header with a custom header of choice, disable the default header.
```

Comment:
Move the assembly introduction here, and put information in a logical order (default behavior, why change, what you can change).

```suggestion
By default, the {product} global header includes a search field, create, support and notification buttons, and a user profile drop-down.
As an administrator, you can configure the {product} global header to create a consistent and flexible navigation bar across your {product-short} instance.
You can extend the global header with additional buttons and customize the order and position of icons and features.
```